tmp/run_sql

```diff
@@ -0,0 +1 @@
+PGPASSWORD=pw psql -f lecture.sql -h pgdb -U postgres postgres
```

Comment:
Similar thing in these 2 command files, `PGPASSWORD=` uses the password you set originally to connect to the DB.

You'll also want to change the `.sql` file to have the name of whatever your file is called.